src/algorithms/spectral/logspectrum.cpp

```diff
@@ -24,68 +24,76 @@ using namespace std;
 using namespace essentia;
 using namespace standard;
 
-const char* LogSpectrum::name = "LogSpectrum";
-const char* LogSpectrum::category = "Spectral";
-const char* LogSpectrum::description = DOC("This algorithm computes spectrum with logarithmically distributed frequency bins. "
-"This code is ported from NNLS Chroma [1, 2]."
-"This algorithm also returns a local tuning that is retrieved for input frame and a global tuning that is updated with a moving average.\n"
-"\n"
-"Note: As the algorithm uses moving averages that are updated every frame it should be reset before  "
-"processing a new audio file. To do this call reset() (or configure())\n"
-"\n"
-"References:\n"
-"  [1] Mauch, M., & Dixon, S. (2010, August). Approximate Note Transcription\n"
-"  for the Improved Identification of Difficult Chords. In ISMIR (pp. 135-140).\n"
-"  [2] Chordino and NNLS Chroma,\n"
-"  http://www.isophonics.net/nnls-chroma");
-
-void LogSpectrum::configure() {
+const char *LogSpectrum::name = "LogSpectrum";
+const char *LogSpectrum::category = "Spectral";
+const char *LogSpectrum::description = DOC("This algorithm computes spectrum with logarithmically distributed frequency bins. "
```

Comment:
For consistency with the majority of the algorithms, we do not do code indentation for the DOC strings (something we need to review in the future).

Reply:
oh yes, you are right. I forgot to switch off the formatter and it modified the code style. No problem, I will resubmit another clean PR and take care of the guidelines.